index.html

```diff
@@ -185,7 +185,7 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
             </ul>
             <button class="article-btn">Get my resume</button>
           </div>
-          <p>
+          <div class="skills-categorie">
             <details open>
               <summary>
                 <span class="summary-title">Langages </span>
```

Comment:
- [ ] Please note that the `border-bottom` is not present after Languages 🔨 Kindly fix this


### **_Mobile version_**

Your Design | Design guideline
--- | ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/762c55b5-565d-4626-b1fe-10e8f4e532f9) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/b1e9fd4f-2245-4238-b4eb-4460e67a8c6d)


### **_Desktop version_**

Your Design | Design guideline
--- | ---|
![image](https://github.com/WilfriedM01/portfolio/assets/106119105/0ffeb657-c8ea-4bbb-9fa9-a2771e3108e9) | ![image](https://github.com/WilfriedM01/portfolio/assets/106119105/8101eada-3322-4ce1-8f9d-ec9cb6c878c0)